Automattic/jetpack
# JITM: regenerate assets

#### Changes proposed in this Pull Request:

* Due to recent dependency updates, the CSS generated by our build tools has changed a bit. Those changes were coming up every time you would `yarn build`.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Not much should change.
* You should still be able to see JITMs properly styled in your dashboard.

#### Proposed changelog entry for your changes:

* N/A
